sk0-front/src/theme/variables.scss

```diff
@@ -179,7 +179,7 @@
     --ion-color-step-900: #e6e6e6;
     --ion-color-step-950: #f2f2f2;
 
-    --ion-item-background: #000000;
+    --ion-item-background: ##1c1c1d;
```

Comment:
This looks like an invalid color to me.

Reply:
but it was a very nice noncolor :-) - corrected now